lib/active_merchant/billing/gateways/reach.rb

```diff
@@ -56,6 +56,32 @@ def capture(money, authorization, options = {})
         commit('capture', post)
       end
 
+      def refund(amount, authorization, options = {})
+        post = {
+          request: {
+            MerchantId: @options[:merchant_id],
+            OrderId: authorization,
+            ReferenceId: options[:reference_id],
+            Amount: amount
+          }
+        }
```

Comment:
Small nit, but we typically follow the pattern of building these hashes, line-by-line like so:

```ruby
request = post[:request] = {}
request[:MerchantId] = @options[:merchant_id]
request[:OrderId] = authorization
...
```

This pattern makes it a bit easier to read (imo) and also helps to break the code into smaller, more manageable methods.

Reply:
This might be better served to touch up in a refactor ticket. Not something that needs to be corrected here, but perhaps something we should consider going forward.

Reply:
Done!